### ISSUE TITLE
Language Filter - 4.1.2 Name, Role, Value - Language dropdown list is not interpreted as expanded or collapsed

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9879]
As a user reliant on the screen reader to interpret form controls, I should be notified on focus and on input whether a dropdown list is expanded or collapsed.

**Linked work item:** 
#9521 

**Expected Result:**
When the user focuses on or selects a drop down list to expand or collapse the list, the screen reader should update he user with the status of the form control (either 'expanded' or 'collapsed').

**Actual Result:**
The screen reader cannot update the user on the status of the languages dropdown either when the form control receives focus or when the user expands or collapses the list.

**Evidence:**

Use NVDA to replicate.

**Comment by apeters** [comment]
Selectwoo doesn't seem to announce the expanded/collapsed state of the dropdown.

**Comment by imogenroberts** [comment]
@apeters should I assume that the language list not providing 'expanded/collapsed' interpretation is expected behaviour? Is there no way to fix this?

**Comment by apeters** [comment]
@imogenroberts I think the only way to "fix" this would be to fork the selectwoo repo and create a fix and submit a PR back to the project, and hopefully they would merge that fix asap.

**Comment by imogenroberts** [comment]
RETEST: Language filter is now interpreted as expanded and collapsed, but collapsing the combobox produces a very long-winded/convoluted screen reader response which may confuse many users. 

**Evidence:**


https://github.com/archesproject/arches/assets/77161085/0e00a238-89eb-4c59-90c6-0e7e976aeed3

**Comment by apeters** [comment]
@imogenroberts I don't know if there's anything we can do about that long response.  Is this something that can be "tuned" with NVDA?  In any case, even with the long response, does this pass?

**Comment by imogenroberts** [comment]
@apeters I don't think there is anything we can do with NVDA to fine-tune its interpretation but having listened to it again this morning I'm more than happy to pass it as it is now.
